insights/parsers/jboss_domain_server_log.py

```diff
@@ -0,0 +1,101 @@
+"""
+JBoss Domain Server Log - File ``$JBOSS_SERVER_LOG_DIR/server.log*``
+====================================================================
+
+Sample input::
+
+
+    16:22:57,476 INFO  [org.xnio] (MSC service thread 1-12) XNIO Version 3.0.14.GA-redhat-1
+    16:22:57,480 INFO  [org.xnio.nio] (MSC service thread 1-12) XNIO NIO Implementation Version 3.0.14.GA-redhat-1
+    16:22:57,495 INFO  [org.jboss.remoting] (MSC service thread 1-12) JBoss Remoting version 3.3.5.Final-redhat-1
+    16:23:03,881 INFO  [org.jboss.as.controller.management-deprecated] (ServerService Thread Pool -- 23) JBAS014627: Attribute 'enabled' in the resource at address '/subsystem=datasources/data-source=ExampleDS' is deprecated, and may be removed in future version. See the attribute description in the output of the read-resource-description operation to learn more about the deprecation.
+    16:23:03,958 INFO  [org.jboss.as.security] (ServerService Thread Pool -- 37) JBAS013371: Activating Security Subsystem
+
+
+Examples:
+    >>> JbossDomainServerlog.filters.append('3.0.14.GA-redhat-1')
+    >>> log = shared[JbossDomainServerlog]
+    >>> log.file_path
+    '/home/test/jboss/machine2/domain/servers/server-one/log/server.log'
+    >>> log.file_name
+    'server.log'
+    >>> error_msgs = log.get('3.0.14.GA-redhat-1')
+    >>> error_msgs[0]['raw_message']
+    16:22:57,476 INFO  [org.xnio] (MSC service thread 1-12) XNIO Version 3.0.14.GA-redhat-1
+    >>> 'Activating Security Subsystem' in log
+    True
+    >>> from datetime import time
+    >>> log.get_after(time(16:23:03))[1]['raw_message']
+    '16:23:03,958 INFO  [org.jboss.as.security] (ServerService Thread Pool -- 37) JBAS013371: Activating Security Subsystem'
+"""
+
+from .. import LogFileOutput, parser
+from insights.specs import jboss_domain_server_log
+import re
+from datetime import time
+
+
+@parser(jboss_domain_server_log)
+class JbossDomainServerlog(LogFileOutput):
```

Comment:
Are there any other `Log` specs should be parsed for "Jboss"? Since these log Parsers are all inherited from "LogFileOutput" and are basically the same, we'd better put all the similar Parsers into one python module. (e.g. foreman_log and nova_log, etc). I mean, before adding other log Parsers, you can just make a base module for Jboss log files, e.g "jboss_log.py".

And BTW, would you mind use upcase "L" for the name "JbossDomainServerLog"?